src/main/java/net/dv8tion/jda/api/entities/Invite.java

```diff
@@ -249,18 +249,15 @@ default String getUrl()
     int getMaxAge();
 
     /**
-    * The max uses of this invite. If there is no limit thus will return {@code 0}.
-    *
-    * <p>This works only for expanded invites and will throw a {@link IllegalStateException} otherwise!
-    *
-    * @throws IllegalStateException
-     *        if this invite is not expanded
-    *
-    * @return The max uses of this invite or {@code 0} if there is no limit
-    *
-    * @see    #expand()
-    * @see    #isExpanded()
-    */
+     * The max uses of this invite.
+     * If there is no limit this will return {@code 0}.
+     *
+     * <p>This works only for expanded invites, and will throw a {@link java.lang.IllegalStateException} otherwise!</p>
+     *
+     * @throws java.lang.IllegalStateException
+     *         if this is not an expanded invite
+     * @return The max uses of this invite or {@code 0} if there is no limit
+     */
```

Comment:
The docs changes here seem unrelated to the rest of this PR, please revert them.